src/libraries/System.Runtime.Serialization.Xml/tests/AssemblyInfo.cs

```diff
@@ -0,0 +1,7 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+[assembly: ActiveIssue("Disabled due to multiple crashes.", TestPlatforms.Windows, TargetFrameworkMonikers.Netcoreapp, TestRuntimes.Mono)]
```

Comment:
@MaximLipnin missed this one during the review, please open an issue for it and replace the message with the link.

Reply:
oh, sorry, I missed it too. I'll update it with an actual issue.

Reply:
`System.Runtime.Serialization.Xml.Tests` has been enabled in https://github.com/dotnet/runtime/pull/34816. No issue is required.